plugins/nomad/target/tgcount.go

```diff
@@ -43,8 +43,12 @@ func (t *NomadGroupCount) Count(config map[string]string) (int64, error) {
 }
 
 func (t *NomadGroupCount) Scale(action strategy.Action, config map[string]string) error {
-	countInt := int(action.Count)
-	_, _, err := t.client.Jobs().Scale(config["job_id"], config["group"], &countInt, &action.Reason, nil, nil, nil)
+	var countIntPtr *int
+	if action.Count != nil {
+		countInt := int(*action.Count)
```

Comment:
ugh, i'm starting to regret using int64, because i'm tired of having to check the safety of these casts. maybe i should change the API in Jobs().Scale() so, at least, you don't have to worry about it.

Reply:
Yeah, it's ugly but it makes sense I guess, since `Count` is an `int` in Nomad.